scripts/generate-framework-agreement-counterpart-signature-pages.py

```diff
@@ -70,6 +70,8 @@
 from dmapiclient import DataAPIClient
 from dmutils.env_helpers import get_api_endpoint_from_stage
 
+AUTOMATED_COUNTERSIGNING_USER_ID = 64041
```

Comment:
Should we document why we've chosen this number?

Reply:
fixed in 4dc6daa